react/features/base/util/interceptComponent.js

```diff
@@ -26,6 +32,17 @@ const _RULES = [
         if (OS === 'android' || OS === 'ios') {
             return UnsupportedMobileBrowser;
         }
+    },
+    () => {
+        if (APP.unsupportedBrowser) {
+            const { isOldBrowser } = APP.unsupportedBrowser;
+
+            if (isOldBrowser) {
+                return UnsupportedDesktopBrowser;
+            }
+
+            return PluginRequiredBrowser;
```

Comment:
I'm probably missing something here, because I don't see where we handle the case of non-old, non-plugin required browser.